app/views/user_mailer/batch_subscription_notification.text.erb

```diff
@@ -3,29 +3,44 @@
-by <%= this_work.pseuds.map{|p| text_pseud(p)}.to_sentence.html_safe %><% end %>
+<% pseud = this_work.pseuds.map{|p| p.name}.to_sentence %>
+<% byline = this_work.pseuds.map{|p| p.byline}.to_sentence %>
+<% title = this_work.title %>
+<% if creation.is_a?(Work) then
+     link  = work_url(this_work)
+     word_count = this_work.word_count
+   else
+     link  = work_chapter_url(this_work, creation)
+     chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe
+   end %>
+<% if creation.is_a?(Work) %>
+<%= if this_work.backdate then
+t('user_mailer.batch_subscription_notification.text.work_backdated', byline: byline, title: title, link: link, word_count: word_count )
+else
+t('user_mailer.batch_subscription_notification.text.work_new', byline: byline, title: title, link: link, word_count: word_count)
+end %>
+<% else %>
+<%= t 'user_mailer.batch_subscription_notification.text.new_chapter_of',byline: byline, title: title, link: link, word_count: creation.word_count, chapter_title: chapter_title %>
+<%end%>
+<%= t 'user_mailer.batch_subscription_notification.text.collected_by', pseud: this_work.pseuds.map{|p| text_pseud(p)}.to_sentence.html_safe %>
 
 <% if !creation.summary.blank? && creation.is_a?(Chapter) %>
-Chapter Summary: <%= raw to_plain_text(sanitize_field(creation, :summary)) %>
+<%= t 'user_mailer.batch_subscription_notification.chapter_summary'%> <%= raw to_plain_text(sanitize_field(creation, :summary)) %>
 <% end %>
 
 <% unless @seen[this_work.id] %>
-Chapters: <%= this_work.chapter_total_display %>
-Fandom: <%= this_work.fandoms.map{|f| f.name}.to_sentence.html_safe %>
-Rating: <%= this_work.rating_string %>
-Warnings: <%= this_work.warning_string %>
-<% if this_work.relationship_string && !this_work.relationship_string.blank? then %>Relationships: <%= this_work.relationship_string %><% end %>
-<% if this_work.character_string && !this_work.character_string.blank? then %>Characters: <%= this_work.character_string %><% end %>
-<% if this_work.freeform_string && !this_work.freeform_string.blank? %>Additional Tags: <%= this_work.freeform_string %><% end %>
-<% if this_work.series.count > 0 %>Series: <%=raw to_plain_text(series_list_for_feeds(this_work)) %><% end %>
+<%= t 'user_mailer.chapters' %> <%= this_work.chapter_total_display %>
+<%= t 'user_mailer.fandom' %> <%= this_work.fandoms.map{|f| f.name}.to_sentence %>
+<%= t 'user_mailer.rating' %> <%= this_work.rating_string %>
+<%= t 'user_mailer.warning' %> <%= this_work.warning_string %>
+<% if this_work.relationship_string && !this_work.relationship_string.blank? then %><%= t 'user_mailer.relationships' %> <%= this_work.relationship_string %><% end %>
+<% if this_work.character_string && !this_work.character_string.blank? then %><%=t 'user_mailer.characters'%> <%= this_work.character_string %><% end %>
```

Comment:
The `then` should be removed, and there should be a spaces added here, like so: `<%= t 'user_mailer.characters '%>`